pkg/flowkit/gateway/emulator.go

```diff
@@ -152,6 +144,20 @@ func convertBlock(block *flowGo.Block) *flow.Block {
 	}
 }
 
+func convertEvents(events []flowGo.Event) []flow.Event {
```

Comment:
I think you could use this method no? https://github.com/onflow/flow-emulator/blob/master/convert/sdk/flow.go#L217

Reply:
oh I totally missed this one, thanks